### PR TITLE
chore(dgraph): Rename gql package to dql

### PR DESCRIPTION
**Pull request by jarifibrahim**
$SUBJECT

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7820)

### CODE REVIEW

**Review by ahsanbarkati — APPROVED** [review]
Looks good. A few comments. Good to merge after fixing the build.

File: dql/parser_test.go [hunk 1]
```diff
@@ -4141,8 +4141,8 @@ func TestMultipleEqual(t *testing.T) {
 
 	gql, err := Parse(Request{Str: query})
```
Comment (ahsanbarkati) [inline dql/parser_test.go:4142]: should be `dql=...`. Build is failing because of this.

File: dql/parser_test.go [hunk 2]
```diff
@@ -4155,7 +4155,7 @@ func TestParseEqArg(t *testing.T) {
 `
 	gql, err := Parse(Request{Str: query})
 	require.NoError(t, err)
-	require.Equal(t, 2, len(gql.Query[0].Filter.Func.Args))
+	require.Equal(t, 2, len(dql.Query[0].Filter.Func.Args))
```
Comment (ahsanbarkati) [inline dql/parser_test.go:4158]: Here as well. Rename gql to dql.

File: dql/parser_test.go [hunk 3]
```diff
@@ -4252,8 +4252,8 @@ func TestParseEqArg2(t *testing.T) {
 `
 	gql, err := Parse(Request{Str: query})
```
Comment (ahsanbarkati) [inline dql/parser_test.go:4253]: Here as well. And all other instances in this file.

**Comment by MichelDiz** [comment]
Closing in favor of #8429

**Pull request closed without merging**